### PR TITLE
feat(quote-block): Add option to add custom icons

### PR DESCRIPTION
**Pull request by rawi96**
This PR adds an option to add custom icons:
https://app.clickup.com/t/2y40cee

### CODE REVIEW

**Review by Walkeezy — APPROVED** [review]

**Review by celine-s — COMMENTED** [review]
Overall: functionality is well implemented and works well. 👍 

The design has a few deviations from the figma file
- Size of the asset should be small. We don't have an assetInput which is size small yet. Needs adaptations in clarify, right?
- divider within dropdown missing: do we have that?
- when selected, the selected icon should itself be the dropdown. The way it is now is probably the easiest (and also a nice) way. Would check it again with Martin. wdyt?

One thought from my side: 
color cannot be adapted to the selected custom icon - do we want to remove the color input when the custom icon is selected? Or do you have another idea?

**Review by celine-s — COMMENTED** [review]
I extracted the quotation handling into a separate component. wdyt?

**Review by SamuelAlev — CHANGES_REQUESTED** [review]

**Review by celine-s — APPROVED** [review]

File: packages/quote-block/src/Quotations.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <div className="tw-flex tw-justify-between tw-gap-x-7">
+                <QuoteBlockIcon {...defaultProps} />
+                {children}
+                <QuoteBlockIcon
+                    {...defaultProps}
+                    customIconId={CUSTOM_ICON_RIGHT_ID}
+                    quoteStyle={quoteStyleRight ?? QuoteStyle.DoubleDown}
+                />
+            </div>
+        )) || <>{children}</>
```
Comment (SamuelAlev) [inline packages/quote-block/src/Quotations.tsx:52]:
```suggestion
        )) : children
```

File: packages/quote-block/src/Quotations.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        color,
+        blockAssets,
+        isCustomSize,
+        sizeValue,
+        sizeChoice,
+        customIconId: CUSTOM_ICON_LEFT_ID,
+        quoteStyle: quoteStyleLeft ?? QuoteStyle.DoubleUp,
+    };
+    return (
+        (isQuotationMarkType && (
```
Comment (SamuelAlev) [inline packages/quote-block/src/Quotations.tsx:42]:
```suggestion
        (isQuotationMarkType ? (
```

File: packages/quote-block/src/settings.tsx [hunk 3]
```diff
@@ -97,19 +107,39 @@ export const settings: BlockSettings = {
...
                 },
                 {
-                    id: 'quoteStyleRight',
+                    id: CUSTOM_ICON_LEFT_ID,
+                    type: 'assetInput',
+                    size: AssetInputSize.Small,
+                    label: 'Custom Icon Left',
+                    extensions: ['svg' as FileExtension.Svg],
+                    objectTypes: [AssetChooserObjectType.ImageVideo],
+                    show: (bundle: Bundle): boolean =>
```
Comment (SamuelAlev) [inline packages/quote-block/src/settings.tsx:123]:
```suggestion
                    show: (bundle) =>
```

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**